lib/ui/home_page/home_charts/syncfusion_line_chart_widget.dart

```diff
@@ -138,9 +138,10 @@ class _SyncfusionLineChartWidgetState extends State<SyncfusionLineChartWidget> {
               xValueMapper: (_ChartData data, _) => data.x,
               yValueMapper: (_ChartData data, _) => data.y1,
               width: 3.5,
+              color: Colors.orange[50], //Make the line transparent
```

Comment:
It would be better to have this as a constant and then use this constant here and where ever else it is used. That way if we change the colour scheme it is a lot easier to change in one place.